package.json

```diff
@@ -206,7 +207,7 @@
     "moment-timezone": "0.5.x",
     "react": "16.x",
     "react-dom": "16.x",
-    "react-intl": "2.x",
+    "react-intl": "2.x || 3.x",
```

Comment:
Are you sure we can make it backwards compatible? Can't we just include it in v10?

Reply:
I think the only real breaking change is the dropping of `intlShape` and the polyfills.

Reply:
Maybe to err on the safe side we should aim this at v10 though...